docs/README.md

```diff
@@ -676,15 +676,112 @@ In this chapter you'll walk through these concepts and its details.
 
 ### 2. Events
 
+Events are **immutable facts** within your application's domain. They are the cornerstone in Booster because of its event-driven and event-sourced nature. Booster events are TypeScript classes decorated with `@Event `:
```

Comment:
```suggestion
Events are **immutable facts** within your application's domain. They are the cornerstone of Booster because of its event-driven and event-sourced nature. Booster events are TypeScript classes decorated with `@Event`:
```